Ghidra/Debug/Framework-Debugging/src/expTypes/c/expTypes.c

```diff
@@ -38,7 +38,7 @@ typedef enum _myenum {
 typedef void (*myfunc_p)(int arg0, long arg1);
 typedef void (*myvargfunc_p)(int arg0, long arg1, ...);
 
-typedef myundef;
+typedef int myundef; // forbidding to be "undefined" in C99 and later
```

Comment:
I'm not sure this is the correct solution here. IIRC, the test (if it still exists) is meant to check for an undefined type here. I think I'd rather delete this file entirely.